tests/test_getting_started.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -eo pipefail
+
+# Step 1: Python environment
+# Check Python version is or later than 3.9
+echo "Step 1: Python environment"
+echo "Checking Python version which should be equal or later than 3.9"
+if ! python -c 'import sys; assert sys.version_info >= (3,9)' > /dev/null; then
+    exit "Python should be 3.9 or later!"
+fi
+
+# Clone repo
+echo "Cloning repo"
+git clone https://github.com/intel/llm-on-ray.git && cd llm-on-ray
+
+# Install dependencies
+echo "Installing dependencies"
+pip install .[cpu] -f https://developer.intel.com/ipex-whl-stable-cpu -f https://download.pytorch.org/whl/torch_stable.html
+
+# Dynamic link oneCCL and Intel MPI libraries
+source $(python -c "import oneccl_bindings_for_pytorch as torch_ccl;print(torch_ccl.cwd)")/env/setvars.sh
+
+# Step 2: Start ray cluster
+ray start --head
+
+# Step 3: Serving
+# take gpt2 for example
+echo "Step 3: Serving"
+echo "Starting ray server for gpt2 with 3 cpu per worker"
+llm_on_ray-serve --config_file .github/workflows/config/gpt2-ci.yaml
+
+# Step 4: Access OpenAI API
+# 1.Using curl
+echo "Step 4: access OpenAI API"
+echo "Way 1: Using curl to access model"
```

Comment:
```suggestion
echo "Method 1: Using curl to access model"
```
pls update the following.